### PR TITLE
Who are we

### PR DESCRIPTION
**Pull request by amitaibu**
#15

Give some background to Gizra.

Link to GitBook - https://www.gitbook.com/book/amitaibu/the-gizra-way/edit#/edit/who-are-we/who_are_we.md

### CODE REVIEW

**Comment by amitaibu** [comment]
@RachelBaram can you take this from here please

**Comment by amitaibu** [comment]
Closing for now

**Comment by RachelBaram** [comment]
@amitaibu - I see there is a conflict with this branch... help.

This is 90% done. I am trying to think about what someone new should probably know about Gizra.  I would love to expand on the "Amitai and Brice story". Get some juicy details. 

Anything you see missing that I can work on until we meet?

File: who_are_we.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+So, if we intersect the Gizra Way with what is going on outside, our purpose becomes clear:
+
+**For Gizra, the world will be a better place if more web projects realize that there is a better way.**
+
+###Our Position
+
+Our purpose probably strikes a chord with anyone who has implementing an IT project of any kind.  We want to reach the clients that can appreciate that there needs to be a better way.  Hence, we believe our message best reaches seasoned executive who have seen enough web projects to get burned. They may have already resigned to the idea that that is how projects are run but know deep down that there has to be another way. Our target clients are most probably in medium sized organizations but really it's about any place a brave executive can successfully influence their organization to adopt a new, simple, and sometimes radical, way of doing things.
+
+**For** medium sized companies
```
Comment (adamhstewart) [inline who_are_we.md]: Why? The UN is the largest  federation of nations is the world (followed maybe by the EU?) Harvard is one of the larger universities. And neither of them are companies. Why would we limit our scope to medium-sized companies when are flag-ship clients are not?

File: who_are_we.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+###Our Personality
+
+Our unusual [About Story](http://www.gizra.com/content/the-about-story/) should get the point across pretty well.
+
+###Our Core Strengths
+
+We love doing projects that challenge us. This naturally led us to excel at the two edges of web development projects:
+
+**Complex Content Management Systems** Gizra was born from here. We joined the Drupal boat pretty early on and had the privilege to pioneered many of the core Drupal modules that serve complex client challenges.  Yes, we are experts here, and it's because we love solving problems.
```
Comment (adamhstewart) [inline who_are_we.md]: consistency with metaphor - either "join the community" or "board the boat" - I prefer the more traditional "join" here.

File: who_are_we.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+**Lean Demonstration of a Product Idea** This can be a prototype, a Minimal Viable Product, or Proof of Concept. The challenge here is how to make the most of a limited budget and short iteration. Our budget driven approach gives us a great advantage here since we know how to define the project well and time-box it both on a project wide level (3-4 months, 1-3 developers) and at the individual task level.
+
+Don't get it wrong, we do and enjoy projects that also sit in the middle of these two edges. Every project lets us practice and teaches us a little more about how we can do better.
+
+###We dabble in:
```
Comment (adamhstewart) [inline who_are_we.md]: Dabble sounds "not serious" - Amitai is maybe the leading Elm evangelist at this point. Jekyll? meh... not sure why it was ever there.

File: who_are_we.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+# Who Are We
+###The Amitai and Brice Story:
+
+A couple of guys, with a pretty interesting perspective, joined forces, synergized, made a lot of mistakes, learned and repeated.  They like the idea of open source, in fact they like the idea of open everything. After plenty of failures and pretty amazing success, these guys, with the help of this team, are still paving the way in the interesting world of web development.
```
Comment (adamhstewart) [inline who_are_we.md]: "repeated" makes it sound like they repeated their mistakes.

File: who_are_we.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+A general problem our industry faces is the high failure rate of web projects due to bad project management. The projects usually do not meet any one of the three criteria of success: being on time, in budget,happy users. 
+
+So, if we intersect the Gizra Way with what is going on outside, our purpose becomes clear:
+
+**For Gizra, the world will be a better place if more web projects realize that there is a better way.**
+
+###Our Position
+
+Our purpose probably strikes a chord with anyone who has implementing an IT project of any kind.  We want to reach the clients that can appreciate that there needs to be a better way.  Hence, we believe our message best reaches seasoned executive who have seen enough web projects to get burned. They may have already resigned to the idea that that is how projects are run but know deep down that there has to be another way. Our target clients are most probably in medium sized organizations but really it's about any place a brave executive can successfully influence their organization to adopt a new, simple, and sometimes radical, way of doing things.
```
Comment (adamhstewart) [inline who_are_we.md]: I think we should avoid words like "executive" because it both excludes non-profits, ed institutions, startups (that only have founders), or anyone with a flat org structure.

Comment (RachelBaram) [inline who_are_we.md]: I am with you... suggestions? We should not not say something because it excludes.

Comment (adamhstewart) [inline who_are_we.md]: I also have a larger argument against "seasoned." I think the way we describe web projects and their tendency for failure hit the chord strongly enough. That is something that will resonate with people who have a lot of experience with web projects, and others who "just get it." Once we say that we are better for people who "have seen enough web projects," we are excluding a set of people who may be very good clients. 

tl;dr - You can get why projects fail without having a lot of web project experience.

File: who_are_we.md (hunk 3 above)
Comment (RachelBaram) [inline who_are_we.md]: Jekyll is there because you wrote it. ;) At first I thought to ask Amitai to write this section but found this sentence from our website. Feel free to update.

File: who_are_we.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+A couple of guys, with a pretty interesting perspective, joined forces, synergized, made a lot of mistakes, learned and repeated.  They like the idea of open source, in fact they like the idea of open everything. After plenty of failures and pretty amazing success, these guys, with the help of this team, are still paving the way in the interesting world of web development.
+
+###The Gizra Way:
+
+The Gizra story is about a way, an outlook, a philosophy that navigates all the internal and external aspects of a web development company. It is knowing and acting on the understanding that *we can do better.*
+
+It is summed up nicely under four sections of The Gizra Way. Let’s look at them:
+
+**Transparency**.  Everything is out in the open, warts and all.  This may be new to you and may take some adjustment. We set the tone as early as our price estimations by basing them on a detailed breakdown of project tasks (small chunks of 12 hours or less) with an attached cost.  This extends into our projects where our clients have full access to Github. They see everything - including our internal communications. Even this book is public.
```
Comment (amitaibu) [inline who_are_we.md]: I think that apart of `Transparency`, the other points have a too markety tone to them that we'll need to tone down. The audience are new employees that need learn about our nerative. For example `Meticulous Execution` is a bit of BS ;) - something like "We try to keep high standards" is probably more realistic.

File: who_are_we.md (hunk 3 above)
Comment (adamhstewart) [inline who_are_we.md]: haha, really? OK, I'll fix.

File: who_are_we.md (hunk 1 above)
Comment (RachelBaram) [inline who_are_we.md]: I added large organizations back in. My concern is that when you work to include everyone, you don't speak to anyone. The 'cut-off' line between who to include and exclude can change and deserves a conversation.  A company's position is usually an 18 month goal. What we have done until now is not necessarily what should or can be done from now on.

File: who_are_we.md (hunk 5 above)
Comment (RachelBaram) [inline who_are_we.md]: "Our message best reaches seasoned executive who have seen enough web projects to get burned" does not mean that we won't or can't reach others who "just get it."  Stating what group we believe our message will reach better will not put off those who "just get it." and it may even have the exclusivity affect that they will want in. I believe narrowing our focus will help us better focus internally and also help us stand out externally.

File: who_are_we.md (hunk 1 above)
Comment (amitaibu) [inline who_are_we.md]: > What we have done until now is not necessarily what should or can be done from now on.

I think this part encapsulates the problem I see with the current PR - it's a bit of mixture of:
1. Out narrative
2. Marketing agenda
3. Future assumptions.

I think we should focus on this chapter only on the narrative. Narrative not necessarily just as an old story, but how we currently or up until now worked.

It might sounds a bit abstract, I don't have an exact vision of how it should look, but I think about the new dev/ account manager coming and trying to understand where did they get to. This is the audience we should focus on. They should know for example:
- How being friendly isn't just a nice to have, but it's part of being professional
- We are friendly, but not "family"
- How we iterate over our success and failues to learn form there
- How we believe in transparency
- Our Thursday toasts
- Why we believe devs should sit around the same table
- How we organize the milk in the frig (a.k.a. Roy's system). It's a bit of a joke obviously, but it shows how the Gizra way is influencing us

**Comment by amitaibu** [comment]
Moving comment to a more prominent place, so it won't be hidden:

> What we have done until now is not necessarily what should or can be done from now on.
> I think this part encapsulates the problem I see with the current PR - it's a bit of mixture of:
1. Out narrative
2. Marketing agenda
3. Future assumptions.
   I think we should focus on this chapter only on the narrative. Narrative not necessarily just as an old story, but how we currently or up until now worked.

It might sounds a bit abstract, I don't have an exact vision of how it should look, but I think about the new dev/ account manager coming and trying to understand where did they get to. This is the audience we should focus on. They should know for example:
- How being friendly isn't just a nice to have, but it's part of being professional
- We are friendly, but not "family"
- How we iterate over our success and failures to learn form there
- How we believe in transparency
- Our Thursday toasts
- Why we believe devs should sit around the same table
- How we organize the milk in the frig (a.k.a. Roy's system). It's a bit of a joke obviously, but it shows how the Gizra way is influencing us

**Comment by amitaibu** [comment]
and just to add to the above. The current text can be probably be used inside more marketing oriented chapters.

**Comment by RachelBaram** [comment]
I moved the text to a new branch: #33 "Our Brand"

@amitaibu  I think you are right that it should be more of a narrative here.  What I wrote is more of an insider's view of our branding. To me, its the same as who we are - what we are on the inside (company culture) equals what we are on the outside (brand) - but I totally 'get it' that it comes off as marketing speak. 

The branding section is super important for the sales team (the fact that there is still disagreement here means there is more work to be done) and kinda important for the account managers.  I would like to suggest that there is added value for the development team to be "let in" on how we are branding ourselves.  IMHO - I would add this to their reading list.

**Comment by amitaibu** [comment]
@RachelBaram completely un-edited but I've started writing down the stuff. We are still as early as 2010 and it's becoming long, so I might divide them to smaller chapters.

**Comment by RachelBaram** [comment]
@amitaibu I really like this content. Would you like me to start editing or wait?

File: who_are_we.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+
+## Theatrical Recognition
+
+Eventhough we were handed a significant module, we were still quite unknown by the international community. Amitai, being the maintainer of Organic groups have indeed started [re-writing](https://www.drupal.org/node/567840) the module and upgrading its architecture in order to make it ready for the, back then, new Drupal 7 release.
+
+A year has passed, Brice had a new baby girl, and Amitai as well, and it was time for DrupalCon Copenhagen. Amitai has submitted a session that was accpeted - a session about the new OG for Drupal 7.
+
+Amitai has decided to take a different approach to the talk. Rather than making a boring presentations about the nitty gritty technical asspects, that is which checkbox or button needs to be clicked, give a more tehatricl sessopn. Amitai and Brice have performed in the first fifteen minutes of the session a completely [imagniry scenario](http://www.gizra.com/content/og7-session-drupalcon-copenhagen/) where Ban Ki Moon, the secretary general of the UN approached Gizra and commisioned them with building sites for all the countries in the world. Through this story they illustrated the new features of OG.
+
```
Comment (bricel) [inline who_are_we.md:53]: imaginary scenario, that became true years later, you should add (UNFPA)

**Comment by amitaibu** [comment]
> Would you like me to start editing or wait?

Wait for now.

**Comment by amitaibu** [comment]
It's good we write these things down, as it actually took me some time to remember if CG was before OS or after :)

File: who_are_we.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+For one year and a half we have worked along with the Nir Yariv, the project manager, who later joined the Gizra team and is now leading some of the most interesting projects we have.
+
+It was our biggest client at the time, paying in dollars and giving us a lot of freedom to constantly iterate and polish our work. We were heavily invested for the first time in automatic testing. The tool we used was bare bone selenium. It was a horrible experience in terms of false positives and taiming the automatic tests, but it showed us what we believed all along. Real platforms can't be built without a certain amount of automatic test coverage.
+
+Even though it was our first international client, we were already work life balance aware. The automatic tests made it possible to maintin it, as the platform hardly broke.
+
+The platform, originally starting in Spanish didn't pick up, so they have decided to translate to English and Portuguese. They required an activity log that could be translated. That was the birth of the [Message ](https://www.drupal.org/project/message) module, later becoming a set of modules know as "the message stack".
```
Comment (RoySegall) [inline who_are_we.md]: `[Message ]` => `[Message]`

**Comment by RachelBaram** [comment]
@amitaibu We need this chapter for our [Training: Being a Gizra Employee](https://docs.google.com/document/d/1qfScE9LKmi3pY1xg6kTEERC7wSw6vfBxYbF6CMyjXMw/edit#heading=h.9siw3gykjegt). First first presentation/training session during the week of November 13th and we are going to ask everyone to read it. Can you have it ready by November 10th? Thank you.

**Comment by amitaibu** [comment]
oh no, time boxing is used against me! 😉 

Sure, I'll do it.

File: who_are_we.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+It's worth mentioning that as much as the scenario was completely ridiculous, in the end of the session one of the people in the audience approached Brice and Amitai and told them he was working in the UN, building an OG site for them - and in fact, their story wasn't so far from reality.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Amitaibu sets a pretty high standard for theatrical presentations.... history in the making! <a href="https://twitter.com/hashtag/drupalcon?src=hash">#drupalcon</a></p>&mdash; Gábor Hojtsy (@gaborhojtsy) <a href="https://twitter.com/gaborhojtsy/status/22181237352">August 26, 2010</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (RachelBaram) [inline who_are_we.md:59]: @amitaibu There is something not right here. It is displayed as text. (row 59)

File: who_are_we.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+Moshe suggested that Amitai take ownership of OG, to which he immediately replied "no." It was a huge responsibility and he felt completely unworthy.  Thirty minutes later, he said "yes, but you should stick around".  
+The next day Amitai already suggested to Moshe a complete re-write of Organic groups.
+
+They returned back to Israel, with commit access to one of the most important Drupal modules. Even back then, they knew this was an important milestone.
+
+
+## Theatrical Recognition
+
+Even though we were handed a significant module, Gizra was still quite unknown by the international community. We did however hire our first employees: an office manager who was also a project manager, and followed by a developer.
```
Comment (RachelBaram) [inline who_are_we.md:48]: @amitaibu Is this correct? I was not sure if it was two employees that you hired.

File: who_are_we.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+
+If you look at our fancy Drupal's installation profile scripts you might not recognize it, but it all started with the work we did a _long_ time ago with CG.
+
+## Harvard University
+
+DrupalCon Denver 2012 arrived, and not long after Entity Reference was born (it started with Damien Tournoud from CG who later [asked Amitai](https://youtu.be/hC4RiK-m0BQ?t=7m40s) to become a co-maintainer) and was followed up with the complete rewrite of Organic groups for Drupal 7. You already read about the complete re-write for OG for Drupal 7? Yes you did. However the 7.1 version had flaws that we wanted to fix. So we went ahead and fixed all the architectural issues we wrongly committed in D7. This 7.2 is still up and running, and is the basis for OG for Drupal 8, that has similar concepts just different code (since Drupal 8 is so different from Drupal 7).
```
Comment (RachelBaram) [inline who_are_we.md:93]: @amitaibu Not sure how DrupalCon Denver 2012 has to do with Entity Reference.

**Comment by RachelBaram** [comment]
@amitaibu Once we close the 3 review items, this is ready to publish. It ends abruptly and I am ok with that. If you have something smart to end the chapter, that is cool too.

File: who_are_we.md (hunk 10 above)
Comment (amitaibu) [reply who_are_we.md:48]: correct.

File: who_are_we.md (hunk 9 above)
Comment (amitaibu) [reply who_are_we.md:59]: once deployed it would look like a tweet

https://www.thegizraway.com/v/702cfd3d5f3b4627110fd82ba753e24a4fd37131/who_are_we.html

(you can see how any revision will look, via for example - https://www.gitbook.com/book/amitaibu/the-gizra-way/build/58244c551781491000d7076b)

File: who_are_we.md (hunk 11 above)
Comment (amitaibu) [reply who_are_we.md:93]: Entity Reference was born a few month before Denver. It's a very important module, so I think people assume it was always there. But it wasn't.

**Comment by amitaibu** [comment]
> if you have something smart to end the chapter, that is cool too.

As time will permit, I'll just finish the chapter properly.

**Comment by RachelBaram** [comment]
> As time will permit, I'll just finish the chapter properly.

We need to publish this chapter by Sunday morning.(Nov 13th, 8:00).  :)

**Comment by amitaibu** [comment]
No, I meant, it's ok for it to end like this (we can add `... to be continued`), and I'll pick it up (after Sunday morning) and finish it.

**Comment by RachelBaram** [comment]
Great. Can you please publish it. Thanks!

**Comment by amitaibu** [comment]
You can too :). Just remove the `WIP`, and click on the Merge pull request
